frontend/package.json

```diff
@@ -8,7 +8,7 @@
     "@formatjs/intl-pluralrules": "^4.0.27",
     "@formatjs/intl-relativetimeformat": "^9.1.6",
     "@formatjs/macro": "^0.2.8",
-    "@hotosm/id": "^2.20.2",
+    "@hotosm/id": "HelNershingThapa/iD.git#osm-auth2.0",
```

Comment:
Can you please describe this change?

Reply:
This uses my [forked iD repo](https://github.com/HelNershingThapa/iD/tree/osm-auth2.0) as the _@hotosm/id_ dependency. I'll send a PR to once we've verified that these changes work as expected.